TrainingTask/Drawer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace TrainingTask
+{
+    public class Drawer
+    {
+        public static object locker = new object();
+
+        public virtual void GenerateElement(int left, int top, char symbol)
```

Comment:
Why method is virtual and nowhere is overriden?

Reply:
Just forgot to fix it. It was previously overrided.